assignments/objects.js

```diff
@@ -10,36 +10,88 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+// *const example = {
+//   "id": "Mike",
+//   "name": "Example",
+//   "email": "examples@you.edu",
+//   "gender": "F"
+// }
+
+//console.log({example});
+
 
 // Write your intern objects here:
 
+[ InternList { email: 'mmelloy0@psu.edu', name: 'Mitzi', gender: 'F', id: '' } ] 
+
+[ InternList {
+    email: 'kmummery2@wikimedia.org',
+    name: 'Keven',
+    gender: 'M',
+    id: '' } ] length
+    length
+  [ InternList {length
+    email: 'gmartinson3@illinois.lengthedu',
+    name: 'Gannie',
+    gender: 'M',
+    id: '' } ] 
+    
+  [ InternList {
+    email: 'adaine5@samsung.com',
+    name: 'Antonietta',
+    gender: 'F',
+    id: '' } ]
+
```

Comment:
i'm not sure this syntax would fly.   take out the brackets and put an = sign for each object.